js/search.js

```diff
@@ -0,0 +1,55 @@
+import { getPosts } from "./main.js";
+import { createButtonsPosts } from "./createButtonsPosts.js";
+import { fetchImages } from "./fetchImages.js";
+
+const buttonSend = document.getElementById("buttonSend");
+const buttonDelete = document.getElementById("deletePostButton");
+
+export function searchPosts(input){
+    if(input == ""){
+        getPosts();
+    }else{
+        fetch("http://localhost:3000/posts")
+        .then(response=>response.json())
+        .then(data=>{
+            const postsList = document.getElementById("listGroup");
+            Array.from(document.getElementById("listGroup").children).forEach(son=>{
```

Comment:
We assume you guys creating an array because you want to iterate over a html collection? 
it can be done with a for loop, or for in loop.